anyway/parsers/injured_around_schools.py

```diff
@@ -227,7 +227,7 @@ def get_injured_around_schools(start_date, end_date, distance):
             not_(and_(SchoolWithDescription.latitude == 0, SchoolWithDescription.longitude == 0)),
             not_(
                 and_(
-                    SchoolWithDescription.latitude == None, SchoolWithDescription.longitude == None
+                    SchoolWithDescription.latitude is None, SchoolWithDescription.longitude is None
```

Comment:
Notice it's not a pythonic operation but SqlAlchemy, from reading this message on Stackoverflow(below) i don't believe it will work

https://stackoverflow.com/a/5632224/13771969